### PR TITLE
show NULL when nullable and default is null

### PR DESCRIPTION
**Pull request by paulr34**
when default value is null and the attribute isNullable then the defaultValue is NULL not empty

### CODE REVIEW

File: src/components/ZclAttributeManager.vue [hunk 1]
```diff
@@ -185,7 +185,12 @@ limitations under the License.
                   : ''
               "
               :disable="isDisabledDefault(props.row.id, selectedCluster.id)"
-              :model-value="defaultValueCheck(props.row.id, selectedCluster.id)"
+              :model-value="
+                props.row.isNullable &&
+                defaultValueCheck(props.row.id, selectedCluster.id) === null
+                  ? 'NULL'
```
Comment (tecimovic) [inline src/components/ZclAttributeManager.vue:191]: I am not happy that we hardcode "NULL" here. This is mixing up the C concept of "NULL" with a concept of "nil" from the spec.

What if we're not generating for C, so we don't have "NULL"?

**Review by tecimovic — APPROVED** [review]

File: src/components/ZclAttributeManager.vue [hunk 2]
```diff
@@ -211,11 +216,6 @@ limitations under the License.
                   color="secondary"
                   label="Null"
                   size="sm"
-                  :disabled="
```
Comment (brdandu) [inline src/components/ZclAttributeManager.vue:214]: Why is this change needed?
